src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private String color;
+
     public String getRandomColor() {
-        return null;
+        int indexColors = new Random().nextInt(Colors.values().length);
+        return Colors.values()[indexColors].toString();
```

Comment:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants